### PR TITLE
Ruletest 6-5 JSON

### PR DESCRIPTION
**Pull request by jugonzal07**
Prepared first cut at ruletest JSON for Section 6 Rule 5.

### CODE REVIEW

**Review by yunjoonjung-PNNL — CHANGES_REQUESTED** [review]
Thank you for doing this, Juan. I verified the testing case (a), (b), and (c)'s floor area was correctly added for applicability check. However, I don't see the proposed part in the testing file. I believe this part is required because there is a part in the RDS saying 
```Get matching space in P_RMR: space_p = match_data_element(P_RMR, Spaces, space_b.id)```. Could you verify this part?

**Comment by yunjoonjung-PNNL** [comment]
Thank you for adding the ```proposed``` parts in the RDS, Juan. I just approved this PR and once this branch is merged, I will test this JSON file with 6-5 code. Thank you!

**Comment by weilixu** [comment]
6-5 is merged by #566

**Pull request closed without merging**
